docs/guides/bake/file-definition.md

```diff
@@ -437,3 +440,6 @@ $ docker buildx bake "https://github.com/tonistiigi/buildx.git#remote-test" "htt
 #8 0.136 -rwxrwxrwx    1 root     root          9620 Jul 27 18:31 vendor.conf
 #8 0.136 /bin/sh: stop: not found
 ```
+
+
+[^1]: See [`docker buildx build` command](https://docs.docker.com/engine/reference/commandline/buildx_build/) reference docs.
```

Comment:
![image](https://user-images.githubusercontent.com/7352848/181205048-b126d57d-48cf-4717-9fd8-8d54c822dbc0.png)

That's a bit weird :confused:

An alternative would be to copy an exact link on each `--flag` to the exact ref in the buildx docs, but I don't think every flag has a section :cry:

Reply:
Yes does not look so great :persevere:

Will change that